ApiDoc/EndpointRouteDescriber.php

```diff
@@ -97,6 +97,10 @@ private function getEndpoint(Route $route)
     {
         $controller = $route->getDefault('_controller');
 
+        if (!\array_key_exists($controller, $this->controllerToRequestMapping)) {
+            return null;
+        }
+
         if (in_array($controller, $this->controllerToRequestMapping)) {
```

Comment:
I believe this block should be removed. The previous condition is a correct replace for that.

Reply:
I also though about removing it, what do you think @sklabinsky and @Dropaq?

Reply:
The statement is simply wrong. First searches by key, second by value. Figure which one you need

Reply:
The second check was removed considering that controller is the key part of the mapping.

So, `$this->controllerToRequestMapping`  receives [`auto1.api_handler.controller_request_mapping`](https://github.com/auto1-oss/service-api-handler-bundle/blob/master/DependencyInjection/CompilerPass/EndpointRouterCompilerPass.php#L83)  parameter which [stores controller part as keys](https://github.com/auto1-oss/service-api-handler-bundle/blob/master/DependencyInjection/CompilerPass/EndpointRouterCompilerPass.php#L100), therefore there is no need of checking values.